### PR TITLE
Latest updates

### PR DESCRIPTION
**Pull request by ramil350**
- Scan API
- Reports layout fix
- Scan/Scoring updates

### CODE REVIEW

**Comment by codelion** [comment]
@RamilGilmanov lets deploy on staging first to confirm the report layout fix.

File: lib/headlines/security_headers/x_frame_options.rb [hunk 1]
```diff
@@ -15,7 +15,7 @@ def score_by_value
       end
 
       def score_by_allow_from
-        whitelisted_domain? ? 1 : 0
+        whitelisted_domain? ? 1 : -1
```
Comment (codelion) [inline lib/headlines/security_headers/x_frame_options.rb:18]: Did we update the scoring for this?

File: spec/lib/headlines/security_headers/x_frame_options_spec.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@ module SecurityHeaders
         context "when value is ALLOW-FROM any domain" do
           let(:value) { "ALLOW-FROM http://example.com" }
 
-          it { is_expected.to eq(0) }
+          it { is_expected.to eq(-1) }
```
Comment (codelion) [inline spec/lib/headlines/security_headers/x_frame_options_spec.rb:41]: Same as above, when did we update this scoring?

**Comment by codelion** [comment]
GTM, except the line notes I left.

File: lib/headlines/security_headers/x_frame_options.rb (hunk 1 above)
Comment (ArthurZaharov) [inline lib/headlines/security_headers/x_frame_options.rb:18]: @codelion 
According to the X-Frame-Options section under HTTP tab in the Scoring Matrix (https://docs.google.com/spreadsheets/d/11egIRbFc2865cFuz5X-o51GmOePQDnexcZY50NYnofU/edit#gid=0), we don't have a score of zero at all. 
So we realized that we had a wrong score initially. This line fixes it.

Comment (codelion) [inline lib/headlines/security_headers/x_frame_options.rb:18]: @ArthurZaharov I think zero score was set by @CalebFenton after doing some tests internally. We may have forgotten to change the scoring sheet. 

@CalebFenton can you confirm what the actual score should be? I think our srcclr.com site doesn't set this header for some particular reason and based on that we had bumped the scoring a bit.

Comment (ArthurZaharov) [inline lib/headlines/security_headers/x_frame_options.rb:18]: @codelion Ok, if so I will revert it (will wait @CalebFenton answer). Thank you.

Comment (CalebFenton) [inline lib/headlines/security_headers/x_frame_options.rb:18]: It's possible I updated scoring elsewhere without updating the scoring spreadsheet, but in this particular case I haven't touched the file (https://github.com/srcclr/security-headers/commits/master/lib/headlines/security_headers/x_frame_options.rb).

I don't mind adjusting the scoring logic (and spreadsheet) in a separate commit.

**Comment by codelion** [comment]
GTM :8ball:

**Pull request merged**